src/components/ChatBody.jsx

```diff
@@ -37,7 +37,7 @@ export default function ChatBody({ socket, selectedChat }) {
 
   return (
     <div className="grid grid-rows-[.1fr_1fr_.115fr] w-full h-screen text-white bg-[#0F0F25]">
-      <ChatBodyHeader />
+      {selectedChat && <ChatBodyHeader selectedChat={selectedChat} />}
```

Comment:
Ist eigentlich doppelt, die Abfrage hast du schon in Chat.jsx gemacht.
Ich würde sie hier aber lassen wollen. Allerdings mit einem if vor dem return und else return NoChatSelected hier im ChatBody. Chat.jsx Return dann einfach nur ChatBody. Ich glaube man muss mit dem {var && <div />} syntax sparsam sein sonst wird's recht schnell unleserlich.

Reply:
Nein ich liebe Frieden 🙂